.npmignore

```diff
@@ -1,9 +0,0 @@
-examples/
-test/
-
-.editorconfig
-.gitingore
-.npmignore
-.travis.yml
-
-CONTRIBUTING.md
```

Comment:
If you have a parent `package.json` you can also specify `private: true` to prevent it getting published by accident